client/src/components/Typeahead/Typeahead.tsx

```diff
@@ -165,13 +165,16 @@ export class Typeahead extends React.Component<TypeaheadProps, TypeaheadState> {
             placeholder={placeholder}
             autoComplete="off"
             value={input_value}
+            type="search"
```

Comment:
Not 100% on this, but I think it should be `type="text"` since it has `role="combobox"`. Have you tried these changes with Mac Voice Over? Problem's always that screen readers can be very inconsistent so it's better to stick as close to the spec/best practices as possible, rather than "whatever works best with Voice Over", but it might help clarify something like this.